etcdserver/metrics.go

```diff
@@ -32,10 +38,36 @@ var (
 		Name: "etcdserver_proposal_failed_total",
 		Help: "The total number of failed proposals.",
 	})
+
+	fileDescriptorInUse = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "file_descriptor_in_use_total",
```

Comment:
use `file descriptor used` because it matches other metric names better.
total is deleted because this is not accumulating count.